app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ComponentType.scala

```diff
@@ -50,6 +51,14 @@ case object Address {
   val fields = (id: FormComponentId) => mandatoryFields(id) ++ optionalFields(id)
 }
 
+object DisplayWidthAttribute extends Enumeration {
```

Comment:
`DisplayWidthAttribute` sound unnecessary long, what about dropping word `Attribute` to stay just with: `DisplayWidth` ?

Reply:
Yes, DisplayWidth looks better